spec/dummy/db/schema.rb

```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```

Comment:
I suppose removal of this comment was because you used a different Ruby version? Could you remove the changes done to `schema.rb` from this PR as they're not really relevant here.